src/components/cookie-consent-drawer.js

```diff
@@ -0,0 +1,29 @@
+import { Button, Drawer } from 'antd';
+import React, { useState } from 'react';
+import cookie from '../common/cookie.js';
+import { useTranslation } from 'react-i18next';
+
+const EDUCANDU_COOKIE_CONSENT = 'educanduCookieConsent';
```

Comment:
Maybe this should be configurable as well

Reply:
Maybe, I don't find it that important. It should maybe align a little bit better with the existing cookie names `SID` and `UILANG` (maybe `COOKIES`?)

Reply:
Cookies are sandboxed to the domain name anyway, so they wouldn't interfere, but anyway, it cannot hurt

Reply:
I've made it an optional thing.